app/tests/spec/views/form.js

```diff
@@ -354,7 +354,9 @@ function (chai, sinon, $, p, FormView, Template, Constants, Metrics, AuthErrors,
 
       it('adds invalid class to the invalid element', function () {
```

Comment:
this needs a `done` callback , + `done()` in the timeout, you can find it in other tests